code/edge/src/androidTest/java/com/adobe/marketing/mobile/util/FunctionalTestHelper.java

```diff
@@ -125,6 +125,20 @@ public void evaluate() throws Throwable {
 		}
 	}
 
+	/**
+	 * Reset the {@link MobileCore} and {@link ServiceProvider} without clearing persistence or database.
+	 * Resets the {@code MobileCore} and {@code ServiceProvider}, sets the {@link FunctionalTestNetworkService}
```

Comment:
nit: can you remove the repetition here e.g:
`Resets the {@code MobileCore} and {@code ServiceProvider} without clearing persistence or database, sets the ...